src/maestro/agents/openai_agent.py

```diff
@@ -26,6 +26,9 @@
 from maestro.agents.agent import Agent as MaestroAgent
 from maestro.agents.openai_mcp import setup_mcp_servers, MCPServerInstance
 
+from dotenv import load_dotenv
+
+load_dotenv()
 
 SUPPORTED_TOOL_NAME: Final[str] = "web_search"
```

Comment:
I don't like how this is fixed. We need to remove and use the flexible tool support @akihikokuroda is adding

CC: @planetf1